app/kubernetes/views/configurations/configmap/create/createConfigMapController.js

```diff
@@ -0,0 +1,170 @@
+import angular from 'angular';
+import _ from 'lodash-es';
+import { KubernetesConfigurationFormValues, KubernetesConfigurationFormValuesEntry } from 'Kubernetes/models/configuration/formvalues';
+import { KubernetesConfigurationKinds } from 'Kubernetes/models/configuration/models';
+import KubernetesConfigurationHelper from 'Kubernetes/helpers/configurationHelper';
+import KubernetesNamespaceHelper from 'Kubernetes/helpers/namespaceHelper';
+import { getServiceAccounts } from 'Kubernetes/rest/serviceAccount';
+import { typeOptions } from '@/react/kubernetes/configs/CreateView/options';
+
+import { confirmWebEditorDiscard } from '@@/modals/confirm';
+import { isConfigurationFormValid } from '../../validation';
+
+class KubernetesCreateConfigMapController {
+  /* @ngInject */
+  constructor($async, $state, $scope, $window, Notifications, Authentication, KubernetesConfigurationService, KubernetesResourcePoolService, EndpointProvider) {
+    this.$async = $async;
+    this.$state = $state;
+    this.$scope = $scope;
+    this.$window = $window;
+    this.EndpointProvider = EndpointProvider;
+    this.Notifications = Notifications;
+    this.Authentication = Authentication;
+    this.KubernetesConfigurationService = KubernetesConfigurationService;
+    this.KubernetesResourcePoolService = KubernetesResourcePoolService;
+    this.KubernetesConfigurationKinds = KubernetesConfigurationKinds;
+
+    this.typeOptions = typeOptions;
+
+    this.onInit = this.onInit.bind(this);
+    this.createConfigurationAsync = this.createConfigurationAsync.bind(this);
+    this.getConfigurationsAsync = this.getConfigurationsAsync.bind(this);
+    this.onResourcePoolSelectionChangeAsync = this.onResourcePoolSelectionChangeAsync.bind(this);
+  }
+
+  onChangeName() {
+    const filteredConfigurations = _.filter(
+      this.configurations,
+      (config) => config.Namespace === this.formValues.ResourcePool.Namespace.Name && config.Kind === this.formValues.Kind
+    );
+    this.state.alreadyExist = _.find(filteredConfigurations, (config) => config.Name === this.formValues.Name) !== undefined;
+  }
+
+  async onResourcePoolSelectionChangeAsync() {
+    try {
+      this.onChangeName();
+      this.availableServiceAccounts = await getServiceAccounts(this.environmentId, this.formValues.ResourcePool.Namespace.Name);
+      this.formValues.ServiceAccountName = this.availableServiceAccounts.length > 0 ? this.availableServiceAccounts[0].metadata.name : '';
+    } catch (err) {
+      this.Notifications.error('Failure', err, 'Unable load service accounts');
```

Comment:
`Unable to load service accounts` instead?

Reply:
updated